src/ocean/util/container/more/Vector.d

```diff
@@ -32,7 +32,7 @@ struct Vector (V, int Size = 0)
 {
         alias add       push;
         alias slice     opSlice;
-        alias push      opCatAssign;
+        public template opOpAssign(string op : "~") { alias opOpAssign = push; }
```

Comment:
Don't we prefer a different way:
```d
public template opOpAssign (string op) if(op == "~") {alias opOpAssign = push;}
```

Reply:
What we want here is not a constrained template but a specifically defined one (all parameters specified).
The compilation time is substantially better with specialisation, the code is shorter and more readable, and the behaviour is the same.